apps/web/src/components/Router.tsx

```diff
@@ -1531,6 +1540,32 @@ const createRoute = (
                 },
               ],
             },
+            {
+              path: "pools/:poolId/screening",
+              element: featureFlags.recordOfDecision ? (
+                <RequireAuth
+                  roles={[ROLE_NAME.PoolOperator, ROLE_NAME.PlatformAdmin]}
+                  loginPath={loginPath}
+                >
+                  <AssessmentEvaluationLayout />
+                </RequireAuth>
+              ) : (
+                <AdminErrorPage />
+              ),
+              children: [
+                {
+                  index: true,
+                  element: (
+                    <RequireAuth
+                      roles={[ROLE_NAME.PoolOperator, ROLE_NAME.PlatformAdmin]}
+                      loginPath={loginPath}
+                    >
+                      <ScreeningAndEvaluationPage />
+                    </RequireAuth>
+                  ),
+                },
+              ],
+            },
```

Comment:
Yeah... I was a bit curious about that too since it seemed like it was going to duplicate linking to process information. I thought maybe the candidate listing thing was supposed to be different so I rolled with it 🤷 

But, based off your two opinions, I rolled it into the existing structure
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8603/commits/6615845670271d086160a31867c72d0b0f84aefc and https://github.com/GCTC-NTGC/gc-digital-talent/pull/8603/commits/de953e4162875968cae941ec6ac763f9dbda919a